modules/local/sra_fastq_ftp.nf

```diff
@@ -29,19 +30,22 @@ process SRA_FASTQ_FTP {
     tuple val(meta), path("*md5")     , emit: md5
 
     script:    
+    def args  = options.args  ? options.args  : '-C - --max-time 600'
+    def args2 = options.args2 ? options.args2 : '12h'
+
     if (meta.single_end) {
         """
-        curl -L ${fastq[0]} -o ${meta.id}.fastq.gz
+        timeout $args2 bash -c 'until curl $args -L ${fastq[0]} -o ${meta.id}.fastq.gz; do sleep 1; done'; echo -e \\\\a

```

Comment:
```suggestion
        timeout $task.time bash -c 'until curl $args -L ${fastq[0]} -o ${meta.id}.fastq.gz; do sleep 1; done'; echo -e \\\\a

```

Reply:
Can we somehow replace `args2` with the NF variable `task.time`? That way the process will timeout according to the max time passed to it via NF and will be quite easy to change via a standard config.

Reply:
`echo -e \\\\a` - what is that doing? 👀

Reply:
thats nice!